packages/flutter/lib/src/material/icon.dart

```diff
@@ -65,6 +65,7 @@ class Icon extends StatelessWidget {
       case Brightness.light:
         return Colors.black;
     }
+    return null;    
```

Comment:
Why do we need a return null in this case?  We've handled all the enum values in the switch.  If the types are correct, this code is dead.